lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1602,6 +1602,67 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
             /* cudnn enabled */ boolFalse);
         return success();
       });
+  patterns.onOp(
+      "MeanVarianceNormalization", 13,
+      [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        Value input;
+        SmallVector<int64_t> axes;
+
+        if (binder.tensorOperand(input) ||
+            binder.s64IntegerArrayAttr(axes, "axes",
+                                       llvm::SmallVector<int64_t>({0, 2, 3})) ||
+            binder.tensorResultType(resultType)) {
+          return failure();
+        }
+        Location loc = binder.getLoc();
+        Value keepDim = rewriter.create<Torch::ConstantBoolOp>(loc, true);
+        Value unBiased = rewriter.create<Torch::ConstantBoolOp>(loc, false);
+        Value none = rewriter.create<Torch::ConstantNoneOp>(loc);
+
+        ArrayRef<int64_t> input_shape = resultType.getSizes();
```

Comment:
You have named this variable as `input_shape` while the value it contains is the result shape.